src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -2601,6 +2607,8 @@ public partial class Exception : System.Runtime.Serialization.ISerializable
         public virtual string StackTrace { get { return default(string); } }
         public System.Reflection.MethodBase TargetSite { get { return default(System.Reflection.MethodBase); } }
         protected event System.EventHandler<System.Runtime.Serialization.SafeSerializationEventArgs> SerializeObjectState { add { } remove { } }
+        [System.Runtime.CompilerServices.FriendAccessAllowedAttribute]
+        internal void AddExceptionDataForRestrictedErrorInfo(string restrictedError, string restrictedErrorReference, string restrictedCapabilitySid, object restrictedErrorObject, bool hasrestrictedLanguageErrorObject = false) { }
```

Comment:
Hey Jan can you explain their purpose? I presume they're not there for unit testing purposes or similar -- how does compiling against S.P.CoreLib fit in here?

---
In reply to: [68819960](https://github.com/dotnet/corefx/pull/9712#discussion_r68819960) [](ancestors = 68819960)